packages/lib/src/Base.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { getApi } from "./api";
+
+  const apiPromise = getApi({
+    clientId: "demo",
+    environment: "dev",
+  });
+</script>
+
+{#await apiPromise}
```

Comment:
Wait for the api to be created.

Reply:
This should be visible on the clients' website. We should look into either not showing anything or provide a "loader placeholder" that has the approximate size of whatever is about to be rendered.

Reply:
I suggest not showing anything while loading

Reply:
Yeah, I'll remove that before it goes to prod :)